main.tf

```
@@ -56,7 +56,7 @@ resource "google_folder_iam_member" "tmp_project_creator" {
 
 module "seed_project" {
   source                      = "terraform-google-modules/project-factory/google"
-  version                     = "~> 10.1.1"
+  version                     = "~> 11"
```

We pin to a major version.

```suggestion
  version                     = "~> 11.2"
```